python2/koans/about_methods.py

```diff
@@ -34,15 +34,15 @@ def test_calling_functions_with_wrong_number_of_arguments(self):
         except Exception as e:
 
             # Note, watch out for parenthesis. They need slashes in front!
-            self.assertMatch(__, e[0])
+            self.assertMatch(r'my_global_function\(\) takes exactly 2 arguments \(3 given\)', e[0])
 
     # ------------------------------------------------------------------
 
     def pointless_method(self, a, b):
         sum = a + b
 
     def test_which_does_not_return_anything(self):
-        self.assertEqual(__, self.pointless_method(1, 2))
+        self.assertEqual(None, self.pointless_method(1, 2))
         # Notice that methods accessed from class scope do not require
```

Comment:
This is important, keep in mind. There's a reason for it but you'll go back to it when you see Classes and Objects.